src/integration/regressionTests/pathnames.js

```diff
@@ -0,0 +1,333 @@
+module.exports = {
```

Comment:
We could potentially pull in the Cypress config so that we don't have to put configuration in multiple places

Reply:
I tried that but some of the Cypress urls were 404ing so I got annoyed and created a new config

Reply:
> Would the tests ever be version controlled? Or just generated on the fly each time & .gitignored?

Yeh we keep them in source control. But when we add a new test we can run the script and it will regenerate all test files with the new test.